### PR TITLE
Enable etcd3-based Jenkins job in 1.5 branch

### PR DESCRIPTION
**Pull request by wojtek-t**
@mml @fgrzadkowski

### CODE REVIEW

**Comment by wojtek-t** [comment]
@fejta @kubernetes/test-infra-maintainers - any hints why bazel failed?

File: jobs/ci-kubernetes-e2e-gce-etcd3-release-1.5.env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### job-env
+JENKINS_PUBLISHED_VERSION=ci/latest-1.5
+GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]
+GINKGO_PARALLEL=y
+PROJECT=k8s-gce-etcd3-1-5
+TEST_ETCD_IMAGE="3.0.14"
+TEST_ETCD_VERSION="3.0.14"
+STORAGE_BACKEND="etcd3"
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gce-etcd3-release-1.5.env]: no quote here in the env files :-)

Comment (fejta) [inline jobs/ci-kubernetes-e2e-gce-etcd3-release-1.5.env]: Hah, great unit test Sen :)

Comment (wojtek-t) [inline jobs/ci-kubernetes-e2e-gce-etcd3-release-1.5.env]: Yeah - the fact that it was caught by test is great (and thanks for pointing that no quotes should be in this file).

**Comment by wojtek-t** [comment]
@fejta @krzyzacy - can one of you please take a look?

**Comment by wojtek-t** [comment]
@krzyzacy - thanks!

Merging to have some results tomorrow and be able to debug if that's what I expected to run :)

**Pull request merged**